src/main/java/com/microsoft/aad/msal4j/ClientApplicationBase.java

```diff
@@ -337,7 +347,9 @@ public T executorService(ExecutorService val) {
 
         /**
          * Sets Proxy configuration to be used by the client application for all network communication.
-         * Default is null and system defined properties if any, would be used.
+         * Default is null and system defined properties if any, would be used. If HTTP client is set on
```

Comment:
I would add that it does set proxy on default http client and if custom http client is set on ....

Reply:
I have tried to clarify further. Let me know if you think it can be further improved.